static/js/sheets/SheetsHomePage.jsx

```diff
@@ -21,8 +20,18 @@ const SheetsHeroBanner = ({title, message, videoOptions, posterImg}) => {
         </div>;
 }
 
+const SheetsSidebar = () => {
```

Comment:
This is another example adding to why we need to refactor how we render the NavSidebar. Another big code repetition here.